subjects/thirdtimeisacharm/README.md

```diff
@@ -0,0 +1,46 @@
+# ThirdTimeIsACharm
+
+### Instructions
+
+Write a function `ThirdTimeIsACharm()` that takes a string and returns another string with every third character.
+
+- Prints the Output followed by newline `\n`.
+- If the string is empty, return newline `\n`.
+- If there is no third character, return `\n`
+
+### Expected function
+
+```go
+func ThirdTimeIsACharm(str string) string {
+}
+```
+
+### Usage
+
+Here is a possible program to test your function:
```

Comment:
Missing a space in between "function" and ":" .

Reply:
it's fixed